plugin-tester-java/src/main/resources/application.conf

```diff
@@ -1 +1,16 @@
 akka.loglevel = INFO
+akka.grpc.client {
+  "helloworld.GreeterService" {
+    host = 127.0.0.1
+    port = 8080
+    override-authority = foo.test.google.fr
+    ssl-config {
+      disabledKeyAlgorithms = [] // Allow weak certificates
+      trustManager {
+        stores = [
+          {path = certs/ca.pem, classpath = true, type = PEM}
```

Comment:
is this a relative path or class path?

Reply:
https://lightbend.github.io/ssl-config/KeyStores.html#configuring-a-trust-manager suggests it's a filesystem path

Reply:
Ah, but you can set a `classpath` boolean to load it from the classpath (https://github.com/lightbend/ssl-config/blob/master/ssl-config-core/src/main/scala/com/typesafe/sslconfig/ssl/Config.scala#L641)

Reply:
You're right, it's a classpath value.